extract.go

```diff
@@ -20,11 +20,15 @@ var (
 
 	// This is for backwards compatability, until we can rip out the old SLA dashboard
 	//[splunkMetrics] 2015/12/21 10:01:37.336610 UUID=08d30fb4-a7b3-11e5-955c-1e1d6de94879 transaction_id=tid_28pbiavoqs publishDate=1450692093737000000 publishOk=true duration=6 endpoint=content
-	pamRegexOLD = regexp.MustCompile(`UUID=([\da-f-]*) transaction_id=(tid_[a-z0-9]*) publishDate=(\d*) publishOk=(\w*) duration=(\d*) endpoint=([\w-]*)`)	//[splunkMetrics] 2015/12/21 10:01:37.336610 UUID=08d30fb4-a7b3-11e5-955c-1e1d6de94879 transaction_id=tid_28pbiavoqs publishDate=1450692093737000000 publishOk=true duration=6 endpoint=content
+	pamRegexOLD = regexp.MustCompile(`UUID=([\da-f-]*) transaction_id=(tid_[a-z0-9]*) publishDate=(\d*) publishOk=(\w*) duration=(\d*) endpoint=([\w-]*)`) //[splunkMetrics] 2015/12/21 10:01:37.336610 UUID=08d30fb4-a7b3-11e5-955c-1e1d6de94879 transaction_id=tid_28pbiavoqs publishDate=1450692093737000000 publishOk=true duration=6 endpoint=content
 
 	//[splunkMetrics] 2015/12/21 10:01:37.336610 UUID=08d30fb4-a7b3-11e5-955c-1e1d6de94879 readEnv=prod-uk transaction_id=tid_28pbiavoqs publishDate=1450692093737000000 publishOk=true duration=6 endpoint=content
 	pamRegex = regexp.MustCompile(`UUID=([\da-f-]*) readEnv=([\w-]*) transaction_id=(tid_[a-z0-9]*) publishDate=(\d*) publishOk=(\w*) duration=(\d*) endpoint=([\w-]*)`)
 
+	//[slaMetrics] 2015/12/21 10:01:37.336610 UUID=08d30fb4-a7b3-11e5-955c-1e1d6de94879 metPublishSla=true okEnvironments=[prod-uk,prod-us] transaction_id=tid_28pbiavoqs publishDate=1450692093737000000
+	//[slaMetrics] 2015/12/21 10:01:37.336610 UUID=08d30fb4-a7b3-11e5-955c-1e1d6de94879 metPublishSla=false okEnvironments=[prod-uk] transaction_id=tid_28pbiavoqs publishDate=1450692093737000000
+	slaPamRegex = regexp.MustCompile(`UUID=([\da-f-]*) metPublishSla=(\w*) okEnvironments=(\[[\w,\-]*\]) transaction_id=(tid_[a-z0-9]*) publishDate=(\d*)`)
```

Comment:
The field in PAM is `metPublishSLA` - which makes more sense to me (since SLA is an acronym)